source/_posts/2023-09-06-release-20239.markdown

```diff
@@ -64,44 +52,47 @@ Don't forget to [join our release party live stream on YouTube](https://www.yout
 
 <lite-youtube videoid="JSNRrWk9IHs" videotitle="Home Assistant 2023.9 Release Party"></lite-youtube>
 
-## Climate, humidifier and water heater more info
+## New climate, humidifier and water heater entity dialogs
 
-The more info for climate, humidifier and water heater entities got an update, if you ask me, they look absolutely stunning!
-They now have a newly designed circular slider, that shows the current temperature and the setpoint(s).
-There are also buttons to change the setpoint.
-It will glow in the middle to show it's current action, as well as show that.
+The entity dialogs for climate, humidifier and water heater entities got an update, if you ask me, it looks absolutely stunning!
+It now has a newly designed circular slider, that shows the current temperature and the target temperature. And allows you to swipe to the desired target temperature easily.
+There are also dedicated buttons to change the target, so it easy to use if you are not using a touchscreen.
+It will glow in the middle to show it's current action, when it is heating or cooling for example.
 <p class='img'>
-<img src='https://media.discordapp.net/attachments/427516175237382144/1146364342015623198/SPOILER_CleanShot_2023-08-30_at_10.png?width=1242&height=1400'></a>
-Temporary screenshot that needs to be replaced.
+<img src='/images/blog/2023-09/climate-card.png'>
+Screenshot of the more info dialog of a climate entity.
 </p>
 
-## A lot of new tile features
+## Lots of new tile features
+
+The tile card gets many new features this release.
 
-A lot of new tile features are added this release:
+The features of a tile card are a great way to bring more functionalities directly into your dashboard, without having the need to open the more info dialog.
 
 - HVAC mode
 - Water heater mode
 - Temperature control
 - Cover position
 - Cover tilt
-- Lawn mower
+- Lawn mower commands
 - Color temperature
 
 <p class='img'>
-<img src='/images/blog/2023-09/tile features.png'></a>
-Temporary screenshot that needs to be replaced.
+<img src='/images/blog/2023-09/tile-features.png'>
+Screenshot of tile cards with the new features.
 </p>
 
 ## Lawn mower
 
 A new entity component was added this release, the lawn mower!
```

Comment:
```suggestion
A new entity type was added to this release: the lawn mower!
```

We general don't refer to it as entity component in the release notes/user end in the past years but as an entity type.